src/main/kotlin/com/example/todolist/global/error/ErrorCode.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.todolist.global.error
+
+enum class ErrorCode(
+    val message : String,
+    val status : Int
+) {
+    // TOKEN
+    EXPIRATION_TOKEN("토큰이 만료됬습니다",403),
+    INVALID_TOKEN("토큰이 유효하지 않습니다",403),
+
+    // AUTH
+    DUPLICATE_EMAIL("이미 존재하는 이메일입니다",400),
```

Comment:
```suggestion
    DUPLICATE_EMAIL("이미 존재하는 이메일입니다",409),
```
존재하는 리소스에 대한 httpStatus는 409 입니다!

Reply:
수정했습니다